lms/djangoapps/commerce/utils.py

```diff
@@ -44,6 +44,13 @@ class EcommerceService(object):
     def __init__(self):
         self.config = CommerceConfiguration.current()
 
+    def toggle_commerce_config(self, enabled=False, checkout_page='/test_basket'):
```

Comment:
`/test_basket` doesn't seem like a useful default outside of testing. Why is this method necessary?